.github/workflows/run_jcb_basic_testing.yaml

```diff
@@ -0,0 +1,87 @@
+name: Run JCB client testing with client changes
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+
+  jcb_integration_tests:
+
+    runs-on: ubuntu-latest
```

Comment:
not sure if this was resolved, but GitHub actions was complaining about `pip` on the latest image

Reply:
I guess since it ran, it's fine

Reply:
Maybe they fixed something?